src/invite/utils.ts

```diff
@@ -98,3 +99,14 @@ export async function createDynamicLink(address: string) {
     },
   })
 }
+
+export async function resolveDynamicLink(link: string) {
```

Comment:
nit: Maybe this should live in a different location? If we use deep links in other features, it might feel weird to pull dynamic link resolution from `invite/utils`.

Reply:
yeah good one, i didn't think about this. the first place that came to mind was to put it with the rest of the firebase functions, let me know if you had something else in mind :)